aten/src/ATen/core/op_registration/op_registration.h

```diff
@@ -879,6 +879,23 @@ class CAFFE2_API Module final {
     impl(name, std::forward<Dispatch>(key), std::forward<Func>(raw_f));
     return std::move(*this);
   }
+  // Convenience overload for unboxed only kernels.  These are quite common
+  // but will be eventually eliminated; this function makes it easy to grep for
+  // them.
+  //
+  // If you're looking how to def or impl with no DispatchKey, use the
+  // CppFunction::makeUnboxedOnly factory directly (the API is compositional;
+  // it's just that we have a LOT of impl calls that are unboxed, so there is
+  // just a little syntax sugar for this case.)
+  template <typename Dispatch, typename Func>
```

Comment:
Can we add a TODO here saying we intend to remove these?

Reply:
sure